wiki/Client/Installation/zh-hk.md

```diff
@@ -28,7 +30,9 @@ outdated: true
 - 享受 osu! 帶來的樂趣！
 
 ###### 添加譜面
-一個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
+
+個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
```

Comment:
that's a chinese character and it means "one", put it back >:(

```suggestion
一個譜面是包含歌曲和播放所需要的文件。包括 .osz 文件。
```

Reply:
TIL